packages/card/src/spectrum-config.js

```diff
@@ -33,7 +33,7 @@ const config = {
                 converter.classToClass('spectrum-Card-content'),
                 converter.classToClass(
                     'spectrum-Card-quickActions',
```

Comment:
nit: lets confirm from css whether there is any change required here or not on the className

Reply:
I informed them of the use of this deprecated component class name and we will be informed if this changes.